lib/aws-efs-creator-stack.ts

```diff
@@ -34,16 +35,19 @@ export class AwsEfsCreatorStack extends cdk.Stack {
     efsSG.applyRemovalPolicy(removalPolicy);
 
     // create an EFS File System
+    const performanceMode = parsePerformanceMode(props.performanceMode);
+    const throughputMode = parseThroughputMode(props.throughputMode);
```

Comment:
**Suggestion:** Add validation to ensure `performanceMode` and `throughputMode` are not undefined before parsing, to prevent runtime errors.
> 🌉  **Medium Priority** 
 > [category: error handling, importance: 7]
```suggestion
    if (!props.performanceMode) {
      throw new Error('Performance mode is not defined.');
    }
    if (!props.throughputMode) {
      throw new Error('Throughput mode is not defined.');
    }
    const performanceMode = parsePerformanceMode(props.performanceMode);
    const throughputMode = parseThroughputMode(props.throughputMode);
```